### ISSUE TITLE
Bareme export: default day

### ISSUE THREAD

**Issue by JeromeLeCoz** [BaRatin-tools/BaRatinAGE#84]
The default day seems to be zero, and clicking on the top arrow yields 2 (instead of 1 as expected). Anyway, 0 does not look like a reasonable day number...

Also, the start year seems to be 2023 instead of 2024. 

And the end date/time could be 2050/01/01 00:00 (as done in Barème for the currently valid curve).

**Comment by IvanHeriver** [comment]
Thanks for catching this bug Jérôme! 
- the zero is indeed not reasonable. I happened only when BaRatinAGE was trying to initialize the day to 1 (so you had to use BaRatinAGE and make a BaReme export on a first of any month (or in january and the bug would have affected the month instead of the day)). It was a difficult bug to solve. but it is solved!
- default dates where the current date for the validity end date and one year before for the validity start date: I modified the code so it is what you suggest: start date is now the current date and end date is 2050/01/01 00:00

The bug should be fixed. It will be part of the next release. I'm closing the issue.
